ckan/Dockerfile

```diff
@@ -4,26 +4,24 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-only
 
-FROM ckan/ckan-base:2.10.5
+FROM ckan/ckan-base:2.11
```

Comment:
important: the same comments done on `Dockerfile.dev` are applied for this file.